### ISSUE TITLE
Change the first page/component that loads when opening the app

### ISSUE THREAD

**Issue by DanDvoracek** [NoDivide/astrum#75]
Hi! First of all thanks for that project, it's great! I love it!

However I'm wondering how to change the first page loaded as I didn't find any way to do it so far.

I have the following in my data.json 

```json
"content": {
        "show_first_page_on_load": true,
```

and yet it is the first component of the list that always appears on (re)load. And this happens even if the history is present in the url (f.e.: .../#introduction --> refers to a page). Any idea how to fix this ? And change the first page that loads to one I chose?

Many thanks anyway, this little app made my day!

**Comment by RyanHavoc** [comment]
@DanDvoracek The `show_first_page_on_load` option lets you control whether your first content page (`intro.md` by default) loads first or your component library. You can't currently control what component appears first... you can control the order of components though so you can just put the components you want to appear first at the top? (<- I may be misunderstanding what you're trying to do...)

**Comment by DanDvoracek** [comment]
Hi @RyanHavoc and thanks for the details. I kinna figured it out already. In my issue I explain that I have the `"show_first_page_on_load": true,` but unfortunately it still loads a component at first load...
The same thing happens on Safari, Firefox and Chrome.

Another thing is that the history is not tracked. If I go to the page introduction for example, le url will change to smth like .../#intro .. If I refresh the page at that stage, it ignores the history and it will reload the "homepage" with the first component.

So a few things going on here actually:

- show_first_page_on_load": true --> simply doesn't work for me (anyone else?)
- the history doesn't seem to be triggered on refresh

Therefore, I'm sorry but I feel like this topic isn't closed.. or maybe there's something I didn't get?

**Comment by DanDvoracek** [comment]
Right, I finally found a little moment to have a look at it and it seems there was a little typo in the js. After having console.logged a couple of things around I found out that there was a reference to an object.array that was missing an "s".

Therefore I ended up creating a new pull request. I hope you don't mind but as it was a tiny thing to fix, I thought it would save you some time !

PR: #77 

Let me know if you're happy to accept it!
